tests/linker/ios/link sdk/Bug1820Test.cs

```diff
@@ -153,6 +154,10 @@ public static DataUpdatesDataUpdateInfo Deserialize (string xml)
 		// note: this also test the linker (5.1+) ability not to remove 'unused' XML setters and .ctors used for serialization
 		public void Bug1820_GenericList ()
 		{
+#if NET && (__IOS__ || __TVOS__)
+			if (Runtime.Arch == Arch.DEVICE)
+				Assert.Ignore ("https://github.com/mono/linker/issues/2266");
```

Comment:
hmm... I thought this only happened on the simulator (JIT)
and the AOT was fine because it used a different option that avoid SRE

your console app test case hits the former (JIT) case